src/PublicApiGenerator/ApiGenerator.cs

```diff
@@ -76,14 +76,14 @@ public static string GeneratePublicApi(this Type[] types, ApiGeneratorOptions? o
         /// <param name="type">The type to generate an API from.</param>
         /// <param name="options">The options to control the API output.</param>
         /// <returns>The API output.</returns>
-        public static string GeneratePublicApi(this Type type, ApiGeneratorOptions? options = null)
+        public static string GeneratePublicApi(this Type type, ApiGeneratorOptions? options)
         {
             (options ??= new ApiGeneratorOptions()).IncludeTypes = new Type[] { type };
             return type.Assembly.GeneratePublicApi(options);
         }
 
         [Obsolete("Use `GeneratePublicApi(this Assembly assembly, ApiGeneratorOptions? options = null)` instead. Will be removed in the next major.")]
```

Comment:
Is this message still clear or should we remove the `= null`?

Reply:
To be consistent we should remove `= null`.